Assets/Scripts/Tank/TankBotStates/ChasingState.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace TankBattle.Tank
+{
+    public class ChasingState : TankBotState
+    {
+        private TankController targetTank;
+        public TankController TargetTank {set { targetTank = value; }}
+        public override void OnEnterState(TankController _tankController, TankView _tankView)
+        {
+            base.OnEnterState(_tankController, _tankView);
+        }
+
+        public override void OnExitState()
```

Comment:
only include override functions if you have some custom code - else they are doing nothing